x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -165,6 +165,8 @@ qualifiedNamePatterns
 identifier
     : UNQUOTED_IDENTIFIER
     | QUOTED_IDENTIFIER
+    | FIRST
+    | LAST
```

Comment:
Is it worth restricting FIRST/LAST to only a function name opposed to any identifier? Although somebody might want to use `| EVAL first=first(column, @timestamp)`. Not sure.